integration-tests/pom.xml

```diff
@@ -136,6 +136,7 @@
         <module>micrometer-mp-metrics</module>
         <module>micrometer-prometheus</module>
         <module>logging-json</module>
+        <module>jaxp</module>
```

Comment:
This module needs to be added to `ci-actions.yml` to be included in the native test workflow. Add it to one of the Misc category (the shortest one and add 5 minutes to the timeout).

Reply:
Done in f3e6f14df24b908373f09a8defa29a424aaa0d0e